### DISCUSSION TITLE
Re: Learning the basics

### DISCUSSION THREAD

**Discussion by slibre2** [community/community#22879] [Other Feature Feedback, Questions, & Ideas]
<p>… if you have the answer , I m interested as well . </p>
<p>I ve made just ONE Github Pages website .I don’t know how I succeed to do that .  I m trying to do a second one , I can’t ! I didn’t use Jekyll because you have to know a specific language . </p>
<p>So all solutions are welcome . </p>

**Comment by angela-crist** [comment, marked as answer]
<p>Hi <a class="mention" href="/slibre2">@slibre2</a>, </p>
<p>Thanks for being part of the GitHub Community Forum! We’re glad you’re here.</p>
<p>The GitHub Learning Lab is a great way to jump into learning about GitHub Pages. </p>
<p><a href="https://lab.github.com/courses?tag=GitHub%20Pages" rel="nofollow noopener">Here’s a link to the GitHub Pages courses.</a></p>
<p>If you have any questions as you are taking the Learning Lab courses, feel free to post to the <a href="/t5/GitHub-Learning-Lab/bd-p/learn">GitHub Learning Lab board</a>. </p>
<p>Best of luck!</p>

**Comment by slibre2** [comment]
<p>Thanks alot Mrs Crist !</p>

**Comment by angela-crist** [comment]
<p>Happy to help!! Enjoy the Learning Lab!</p>

**Comment by jhvanderschee** [comment]
<p>You can try the basic tutorial from JekyllCodex.</p>
<p><a href="https://jekyllcodex.org/getting-started/" rel="nofollow noopener">https://jekyllcodex.org/getting-started/</a></p>
